### PR TITLE
Fix deserialization of UpdatePFS message

### PR DESCRIPTION
**Pull request by palango**
Fixes https://github.com/raiden-network/raiden-services/issues/158

### CODE REVIEW

File: raiden/messages.py [hunk 1]
```diff
@@ -1830,7 +1832,7 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'type': self.__class__.__name__,
             'chain_id': self.chain_id,
             'nonce': self.nonce,
-            'token_network_address': to_normalized_address(self.token_network_address),
+            'token_network_address': to_checksum_address(self.token_network_address),
```
Comment (LefterisJP) [inline raiden/messages.py]: hmm this is not looking good. Every other message we send around exports to the `normalized_address`. Why should this message be different? I would instead adjust the PFS and not the client here.

File: raiden/tests/unit/test_messages.py [hunk 2]
```diff
@@ -148,6 +148,8 @@ def test_update_pfs():
     message.sign(signer)
     assert recover(message._data_to_sign(), message.signature) == ADDRESS
 
+    assert message == UpdatePFS.from_dict(message.to_dict())
```
Comment (LefterisJP) [inline raiden/tests/unit/test_messages.py]: nice, was about to ask for a serialization test but here it is

File: raiden/messages.py (hunk 1 above)
Comment (palango) [reply raiden/messages.py]: Is there a reason not to use checksum addresses? Consistency is nice, but maybe there isn't a conscious thought behind using normalized aka lower case hex addresses.

Comment (LefterisJP) [reply raiden/messages.py]: Can't say for sure. But the only reason for this kind of checksumming is to protect the user from mistyping. So it serves no purpose in the wire between raiden clients.

Comment (palango) [reply raiden/messages.py]: A checksum might always be useful, but I don't really care. Changed it.

File: raiden/messages.py [hunk 3]
```diff
@@ -1848,11 +1849,12 @@ def from_dict(
             nonce=data['nonce'],
             transferred_amount=data['transferred_amount'],
             locked_amount=data['locked_amount'],
```
Comment (konradkonrad) [inline raiden/messages.py]: according to RFC 7159, the range for numbers in JSON (which you're ultimately serializing to) is  `[-(2**53)+1, (2**53)-1]`. So for token values, you might want to serialize as `str` and deserialize an `int` as we do elsewhere, e.g. https://github.com/raiden-network/raiden/blob/master/raiden/messages.py#L1583

Quote from the RFC:
> Note that when such software is used, numbers that are integers and
   are in the range [-(2**53)+1, (2**53)-1] are interoperable in the
   sense that implementations will agree exactly on their numeric
   values.

**Pull request merged**